### PR TITLE
AMMP 4144: save slave address change for holykell h5620

### PR DESCRIPTION
**Pull request by tienthai-ammp**

### CODE REVIEW

**Comment by tienthai-ammp** [comment]
@svet-b here is the link to the task with the document https://ammpio.atlassian.net/browse/AMMP-4144?atlOrigin=eyJpIjoiMjE3OWQxMmU3MDNmNDMxMmI4OTVlNzkyN2EzNmMwZmEiLCJwIjoiaiJ9
not sure what value is expected (in the `mod.write_register(registeraddress, value, number_of_decimals, functioncode)` function call), but given that it's not mentioned, I assume it's not important and use "1" for now. Do you have some idea if the value param is considered when running the save change command?

File: src/node_mgmt/commands.py [hunk 1]
```diff
@@ -180,14 +180,17 @@ def _set_address_holykell(mod: minimalmodbus.Instrument, result: dict,
         try:
             mod.write_register(80, target_slave_id, 0, 6)
             sleep(1)
+            # command to save changes
+            mod.address = target_slave_id
+            mod.write_register(64, 49087, 0, 6)
```
Comment (svet-b) [inline src/node_mgmt/commands.py:185]: Shouldn't it be register 78 (not 64)? Also where does 49087 come from? I think the value should just be 1 (if we follow the example in the docs)

Comment (svet-b) [reply src/node_mgmt/commands.py:185]: Oh ok I think I kind of see - there's another save command that we have lower down, which has now been moved up here. And we use the new approach only for the "alternative" model.

**Comment by svet-b** [comment]
Yeah I suppose this is right. In their example they state
> For example: changed id from 01 to 02, then you should input 02 06 00 4E 00 01 28 2E
(attention 28 2E is CRC)

Indeed in that payload the value they show is `1` so we can just use that. (if you want, check out the "RTU (hex)" column in the first table in https://www.modbustools.com/modbus.html to see how to interpret a Modbus payload)

**Review by svet-b — APPROVED** [review]

File: src/node_mgmt/commands.py (hunk 1 above)
Comment (tienthai-ammp) [reply src/node_mgmt/commands.py:185]: yeah I'm not sure why we have that number, I checked the old doc with Olu and we also couldn't figure it out (but that doc doesn't have any info about how to change slave address anw) and this save part was done right after the one where we only do write_register to register `80`

**Pull request merged**